### PR TITLE
New Fonts for Paper

### PR DESCRIPTION
**Pull request by Shadowmech88**
Adds font face tags when writing with a pen, with support for a number of different styles.
Renames [small] to [tiny] and adds a new [small] tag that makes the font a bit bigger than the previous one, but still smaller than the default font.
Adds font tag usage to the help menu for paper and nanopaper.
List of the new fonts:
fonts

### CODE REVIEW

**Comment by ririchiyo** [comment]
http://www.html-5.com/changes/deprecated/index.html

**Comment by Shadowmech88** [comment]
The game uses HTML5? Also the tags did work in-game when I tested it.

**Comment by PJB3005** [comment]
Nah, BYOND doesn't use HTML 5, but I'd still not use tags that aren't present in HTML 5 anymore.

**Comment by ririchiyo** [comment]
BYOND does use HTML5 for the web client of BYOND which is likely what will continue to be developed in the future with the actual application being phased out.

File: code/modules/paperwork/pen.dm [hunk 1]
```diff
@@ -139,10 +139,32 @@ var/paperwork_library
...
 	addReplacement(REG_BBTAG("list"), "<ul>")
 	addReplacement(REG_BBTAG("/list"), "</ul>")
+	addReplacement(REG_BBTAG("agency"),  	"<span style=\"font-family:Agency FB\">")
+	addReplacement(REG_BBTAG("/agency"), 	"</span>")
+	addReplacement(REG_BBTAG("algerian"),  	"<span style=\"font-family:Algerian\">")
+	addReplacement(REG_BBTAG("/algerian"), 	"</span>")
+	addReplacement(REG_BBTAG("arial"),  	"<span style=\"font-family:Arial\">")
+	addReplacement(REG_BBTAG("/arial"), 	"</span>")
+	addReplacement(REG_BBTAG("arialb"),  	"<span style=\"font-family:Arial Black\">")
+	addReplacement(REG_BBTAG("/arialb"), 	"</span>")
```
Comment (ComicIronic) [inline code/modules/paperwork/pen.dm]: You know what annoys me about this?
It isn't true.
The equivalent of a font break isn't a span end, it's that specific span end.
This makes 
`[arialb][small]This is small arial bold[/arialb] but this isn't small[/small]`
That doesn't make any sense.

Comment (Shadowmech88) [inline code/modules/paperwork/pen.dm]: Well, I'm not sure there's a way around that, unfortunately. I can't find any HTML tags that are both differentiable, and don't make a line break after the tag end. In fact <span> was the only tag I could find that did not, in and of itself, change the text inside it in any way by default. Is there any way to differentiate <span> tags from one another?

**Comment by d3athrow** [comment]
CHANGELOG

**Pull request merged**